src/openrct2/scripting/bindings/ride/ScRide.cpp

```diff
@@ -524,6 +524,23 @@ namespace OpenRCT2::Scripting
         return ride != nullptr ? ride->GetRideTypeDescriptor().LiftData.minimum_speed : 0;
     }
 
+    uint8_t ScRide::satisfaction_get() const
+    {
+        auto ride = GetRide();
+        return ride != nullptr ? ride->satisfaction : 0;
+    }
+
+    void ScRide::satisfaction_set(uint8_t value)
```

Comment:
Since the ride satisfaction is something that can very well be updated multiple times per tick, I do not see any sense of making this writable.